### ISSUE TITLE
Panels should have accessible label

### ISSUE THREAD

**Issue by karlgroves** [jsbin/jsbin#1923]
Like all form elements, `textarea` elements also require labels.

Unfortunately the way CodeMirror works is to basically hide away the `textarea` and replace it with a series of divs.  

In the past, I've added the label after-the-fact after the CodeMirror has been created by adding `aria-label` to the control.

Ultimately this is something CodeMirror should be doing correctly, but in the short term jsbin could add this themselves.
